jobclient/src/main/java/com/twosigma/cook/jobclient/Job.java

```diff
@@ -276,26 +281,26 @@ public Builder addLabel(String key, String value) {
 
         /**
          * Adds a collection of labels to the job.
-         *
+         * <p>
          * This adds the labels to the job; it won't remove labels that were previously set.
          *
          * @param labels specifies the labelironment to add to this job.
          * @return this builder
          */
-        public Builder addLabels(Map<String,String> labels) {
+        public Builder addLabels(Map<String, String> labels) {
             _labels.putAll(labels);
             return this;
         }
 
         /**
          * Adds a collection of label to the job.
-         *
+         * <p>
          * This resets the labels of the job; it will remove labels that were previously set.
          *
          * @param labels specifies the labels to set for this job.
          * @return this builder
          */
-        public Builder setLabels(Map<String,String> labels) {
+        public Builder setLabels(Map<String, String> labels) {
             _labels = ImmutableMap.copyOf(labels);
```

Comment:
I know this isn't part of this PR, but a line in this method changed. But, since `_labels` is now an immuatble map, doesn't this mean that calls to `addLabels` will now fail? Do we have a unit test that covers this case? I'm okay if this is an issue and covered in a separate PR.

Reply:
I will address it in different PR